pygitguardian/client.py

```diff
@@ -538,6 +541,71 @@ def multi_content_scan(
 
         return obj
 
+    def scan_and_create_incidents(
+        self,
+        documents: List[Dict[str, str]],
+        source_uuid: UUID,
+        extra_headers: Optional[Dict[str, str]] = None,
+        params: Optional[Dict[str, Any]] = None,
+    ) -> Union[Detail, MultiScanResult]:
```

Comment:
Do we need that `params` argument? I think it would be better to have properly typed optional arguments than a pass-through like this.

Also, can you mark optional arguments as keyword-only by adding a `*` before `extra_headers` so that we can safely reorder them if need be?